src/Microsoft.DotNet.Build.Tasks/GenerateResourcesCode.cs

```diff
@@ -169,6 +174,18 @@ private void WriteDebugCode()
             }
         }
 
+        private void WriteGetTypeProperty()
+        {
+            if (_targetLanguage == TargetLanguage.CSharp)
+            {
+                _targetStream.WriteLine("        internal static Type ResourceType {1}{3}              get {1} return typeof({0}); {2}{3}        {2}", _resourcesName + ".SR", "{", "}", Environment.NewLine);
```

Comment:
Where does this get used? I don't see where you are generating the code that calls this.